rxdiffadapter/src/main/java/com/revolut/rxdiffadapter/RxDiffAdapter.kt

```diff
@@ -141,9 +143,18 @@ open class RxDiffAdapter @Deprecated("Replace with constructor without delegates
             diffResult.dispatchUpdatesTo(this)
         }
 
+        val lastCallback = lastDispatchDiffCallback
+        if (lastCallback != null) {
+            rv.removeCallbacks(lastCallback)
```

Comment:
Cancelling current task may postpone dispatching items to recycler(if updates happens frequently). 
Proposal here:
- let's schedule task for dispatching new state(if it's required)
- make that runnable mutable and update it's state here

Reply:
Will do in a separate ticket and dm you with the reference